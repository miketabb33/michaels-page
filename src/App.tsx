import React from 'react'
import Router from './router/Router'
import GlobalStyle from './styles/GlobalStyles'
import { ThemeContextProvider } from './context/ThemeContext'
import { AuthContextProvider } from './context/AuthContext'
import { StyledProvider } from './context/StyledProvider'

const App = () => (
  <React.StrictMode>
    <AuthContextProvider>
      <ThemeContextProvider>
        <StyledProvider>
          <GlobalStyle />
          <Router />
        </StyledProvider>
      </ThemeContextProvider>
    </AuthContextProvider>
  </React.StrictMode>
)

export default App
