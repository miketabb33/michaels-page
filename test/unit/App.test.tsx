import React from 'react'
import App from '../../src/App'
import { shallowRenderer } from '../shallowRenderer'

describe('App', () => {
  it('should match snapshot', () => {
    const renderer = shallowRenderer(<App />)
    expect(renderer).toMatchSnapshot()
  })
})
