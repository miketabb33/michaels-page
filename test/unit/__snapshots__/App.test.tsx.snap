// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<UNDEFINED>
  <AuthContextProvider>
    <ThemeContextProvider>
      <StyledProvider>
        <Memo(l) />
        <Router />
      </StyledProvider>
    </ThemeContextProvider>
  </AuthContextProvider>
</UNDEFINED>
`;
